docs/spec.md

```diff
@@ -1918,7 +1918,8 @@ If `feature_group_count = 1` and `batch_group_count = 1`, then for all
   * `padded_lhs = pad(lhs, 0, lhs_padding[:, 0], lhs_padding[:, 1], lhs_base_dilations)`.
   * `lhs_window_start = lhs_shape(0, output_spatial_index, 0) * lhs_window_strides`.
   * `lhs_window = slice(padded_lhs, lhs_window_start, lhs_window_start + lhs_window_dimensions, lhs_window_dilations)`.
-  * `dot_product = dot_general(lhs_window, rhs,
+  * `reversed_rhs = reverse(rhs, [kernel_spatial_dimensions[dim] for dim in [0, size(window_reversal) and window_reversal[dim] = true])`.
```

Comment:
Given that the corresponding input is called `window_reversal`, let's specifically reverse the window? I don't think it matters whether we reverse lhs or rhs, so the proposed formalism is mathematically correct too, but I think it'll read better if it'll be `reversed_lhs_window` and `rhs` rather than the other way around.